Unity/2019.1/Events with Caliper/Assets/Scripts/CaliperEventManager.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Net.Http;
+using UnityEngine;
+using UnityEngine.UI;
+using ImsGlobal.Caliper;
+using ImsGlobal.Caliper.Entities.Agent;
+using ImsGlobal.Caliper.Entities.Media;
+using ImsGlobal.Caliper.Events;
+using ImsGlobal.Caliper.Events.Media;
+using NodaTime;
+
+public class CaliperEventManager : MonoBehaviour
+{
+    private string endpointID;
+
+    private string deviceID;
+
+    public Button actionButton;
+    public Button testPushButton;
+
+    private static readonly HttpClient client = new HttpClient();
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        deviceID = SystemInfo.deviceUniqueIdentifier; // unique for every device
+
+        actionButton.onClick.AddListener(ActionButtonEvent);
+        testPushButton.onClick.AddListener(TestPushButtonEventAsync);
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+
+    }
+
+    void ActionButtonEvent()
+    {
+        TestCaliperEventAsync();
+    }
+
+    async void TestCaliperEventAsync() // should not return void
```

Comment:
If we are not able to send the caliper event using the library then this code needs appropriate comments works from Unity Simulator and not from the Device

Reply:
I have added comments.